core/types/receipt.go

```diff
@@ -30,7 +30,7 @@ import (
 //go:generate gencodec -type Receipt -field-override receiptMarshaling -out gen_receipt_json.go
 
 var (
-	receiptStatusFailedRLP     = []byte{}
+	receiptStatusFailedRLP     []byte
```

Comment:
This is a constant that's going to go into the RLP encoder (not sure if it can end up on json interfaces). Encoders may treat `nil` and `[]byte{}` differently (json does). Let's not change this (perhaps add a comment to avoid future issues).

Reply:
Sure, I will close this PR and open up one to add that comment.